packages/zipkin/src/tracer/index.js

```diff
@@ -196,16 +196,10 @@ class Tracer {
     return this._localEndpoint;
   }
 
-  recordAnnotation(annotation, timestamp = now(this._startTimestamp, this._startTick)) {
```

Comment:
before, even when we didn't sample, we incurred a timestamp cost. now, it is deferred